app/views/groups/show.html.erb

```diff
@@ -0,0 +1,118 @@
+<div class="uk-height-viewport uk-responsive-width">
+  <div class="uk-container-center">
+    <div class="uk-panel uk-panel-box">
+      <h3 class="uk-panel-title"><%= @group.name %></h3>
+      <h2 class="uk-h2">Kör adatai</h2>
+      <table>
+        <tbody>
+          <tr>
+            <th class="group-attr">Név</th>
+            <td><%= @group.name %></td>
+          </tr>
+          <tr>
+            <th class="group-attr">Alapítási éve</th>
+            <td><%= @group.founded %></td>
+          </tr>
+          <tr>
+            <th class="group-attr">Svie tag</th>
+            <td><%= @group.issvie ? "Igen" : "Nem" %></td>
+          </tr>
+          <tr>
+            <th class="group-attr">Weboldal</th>
+            <td><%= @group.webpage %></td>
+          </tr>
+          <tr>
+            <th class="group-attr">Levelezőlista</th>
+            <td><%= @group.maillist %></td>
+          </tr>
+          <tr>
+            <th class="group-attr">Bemutatkozás</th>
+            <td><%= @group.description %></td>
+          </tr>
+        </tbody>
+      </table>
+      <% if @group.users_can_apply && !@own_membership %>
+        <%= form_tag group_membership_index_path(@group), {method: 'post'} do %>
+          <%= button_tag "Jelentkezés körbe", {class: "uk-button uk-button-primary"} %>
+        <% end %>
+      <% end %>
+      <h2 class="uk-h2">Aktív tagok</h2>
+      <table class="uk-table uk-table-striped uk-table-hover">
+        <thead>
+          <tr>
+            <th>Név</th>
+            <th>Becenév</th>
+            <th>Betöltött poszt</th>
+            <th>Tagság kezdete</th>
+            <% if is_leader %>
+              <th>Posztok állítása</th>
+              <th>Öregtaggá avatás</th>
+              <th>Törlés</th>
+            <% end %>
+          </tr>
+        </thead>
+        <tbody>
+          <% @group.memberships.select{ |m| !m.end }.each do |membership| %>
+          <% member = membership.user %>
+          <tr>
+            <td><%= member.lastname %> <%= member.firstname %></td>
+            <td><%= member.nickname %></td>
+            <td>
+              <%= membership.post_types.map(&:pttip_name).join(', ') %>
+            </td>
+            <td><%= membership.start %></td>
+            <% if is_leader %>
+              <td></td>
+              <td>
+                <%= link_to "Öreggé avatás", group_membership_inactivate_path(@group.id, membership.id),
```

Comment:
You should use the `button_to` helper here which generates a form with a button. Then you can style it in a way that works for you. You should not use GET method for destructive things. See my comment for the associated routes.